
[34m[1mwandb[39m[22m: [33mWARNING[39m The save_model argument by default saves the model in the HDF5 format that cannot save custom objects like subclassed models and custom layers. This behavior will be deprecated in a future release in favor of the SavedModel format. Meanwhile, the HDF5 model is saved as W&B files and the SavedModel as W&B Artifacts.
Epoch 1/30

108/111 [============================>.] - ETA: 0s - loss: 0.6914 - mean_absolute_error: 0.1367
/home/jean-sebastien/Documents/s7/PMC/venv/lib/python3.11/site-packages/keras/src/engine/training.py:3103: UserWarning: You are saving your model as an HDF5 file via `model.save()`. This file format is considered legacy. We recommend using instead the native Keras format, e.g. `model.save('my_model.keras')`.

111/111 [==============================] - ETA: 0s - loss: 0.6913 - mean_absolute_error: 0.1363
111/111 [==============================] - 11s 64ms/step - loss: 0.6913 - mean_absolute_error: 0.1363 - val_loss: 0.6848 - val_mean_absolute_error: 0.1198
Epoch 2/30
105/111 [===========================>..] - ETA: 0s - loss: 0.6549 - mean_absolute_error: 0.0855
/home/jean-sebastien/Documents/s7/PMC/venv/lib/python3.11/site-packages/keras/src/engine/training.py:3103: UserWarning: You are saving your model as an HDF5 file via `model.save()`. This file format is considered legacy. We recommend using instead the native Keras format, e.g. `model.save('my_model.keras')`.
  saving_api.save_model(
111/111 [==============================] - 4s 39ms/step - loss: 0.6541 - mean_absolute_error: 0.0835 - val_loss: 0.6373 - val_mean_absolute_error: 0.0446
Epoch 3/30

 97/111 [=========================>....] - ETA: 0s - loss: 0.6284 - mean_absolute_error: 0.0416
/home/jean-sebastien/Documents/s7/PMC/venv/lib/python3.11/site-packages/keras/src/engine/training.py:3103: UserWarning: You are saving your model as an HDF5 file via `model.save()`. This file format is considered legacy. We recommend using instead the native Keras format, e.g. `model.save('my_model.keras')`.
  saving_api.save_model(

111/111 [==============================] - 4s 39ms/step - loss: 0.6286 - mean_absolute_error: 0.0410 - val_loss: 0.6338 - val_mean_absolute_error: 0.0362
Epoch 4/30
 80/111 [====================>.........] - ETA: 0s - loss: 0.6270 - mean_absolute_error: 0.0366
/home/jean-sebastien/Documents/s7/PMC/venv/lib/python3.11/site-packages/keras/src/engine/training.py:3103: UserWarning: You are saving your model as an HDF5 file via `model.save()`. This file format is considered legacy. We recommend using instead the native Keras format, e.g. `model.save('my_model.keras')`.

109/111 [============================>.] - ETA: 0s - loss: 0.6267 - mean_absolute_error: 0.0363
111/111 [==============================] - 5s 41ms/step - loss: 0.6267 - mean_absolute_error: 0.0363 - val_loss: 0.6329 - val_mean_absolute_error: 0.0346
Epoch 5/30
 61/111 [===============>..............] - ETA: 1s - loss: 0.6282 - mean_absolute_error: 0.0341
/home/jean-sebastien/Documents/s7/PMC/venv/lib/python3.11/site-packages/keras/src/engine/training.py:3103: UserWarning: You are saving your model as an HDF5 file via `model.save()`. This file format is considered legacy. We recommend using instead the native Keras format, e.g. `model.save('my_model.keras')`.

109/111 [============================>.] - ETA: 0s - loss: 0.6260 - mean_absolute_error: 0.0340
111/111 [==============================] - 4s 41ms/step - loss: 0.6259 - mean_absolute_error: 0.0340 - val_loss: 0.6323 - val_mean_absolute_error: 0.0322
Epoch 6/30

109/111 [============================>.] - ETA: 0s - loss: 0.6252 - mean_absolute_error: 0.0320
/home/jean-sebastien/Documents/s7/PMC/venv/lib/python3.11/site-packages/keras/src/engine/training.py:3103: UserWarning: You are saving your model as an HDF5 file via `model.save()`. This file format is considered legacy. We recommend using instead the native Keras format, e.g. `model.save('my_model.keras')`.
  saving_api.save_model(
111/111 [==============================] - 4s 40ms/step - loss: 0.6253 - mean_absolute_error: 0.0320 - val_loss: 0.6317 - val_mean_absolute_error: 0.0299
Epoch 7/30

107/111 [===========================>..] - ETA: 0s - loss: 0.6253 - mean_absolute_error: 0.0299
/home/jean-sebastien/Documents/s7/PMC/venv/lib/python3.11/site-packages/keras/src/engine/training.py:3103: UserWarning: You are saving your model as an HDF5 file via `model.save()`. This file format is considered legacy. We recommend using instead the native Keras format, e.g. `model.save('my_model.keras')`.
  saving_api.save_model(
111/111 [==============================] - 4s 39ms/step - loss: 0.6247 - mean_absolute_error: 0.0298 - val_loss: 0.6312 - val_mean_absolute_error: 0.0281
Epoch 8/30

 92/111 [=======================>......] - ETA: 0s - loss: 0.6248 - mean_absolute_error: 0.0269
/home/jean-sebastien/Documents/s7/PMC/venv/lib/python3.11/site-packages/keras/src/engine/training.py:3103: UserWarning: You are saving your model as an HDF5 file via `model.save()`. This file format is considered legacy. We recommend using instead the native Keras format, e.g. `model.save('my_model.keras')`.

110/111 [============================>.] - ETA: 0s - loss: 0.6242 - mean_absolute_error: 0.0267
111/111 [==============================] - 4s 41ms/step - loss: 0.6241 - mean_absolute_error: 0.0267 - val_loss: 0.6307 - val_mean_absolute_error: 0.0252
Epoch 9/30

101/111 [==========================>...] - ETA: 0s - loss: 0.6241 - mean_absolute_error: 0.0250
/home/jean-sebastien/Documents/s7/PMC/venv/lib/python3.11/site-packages/keras/src/engine/training.py:3103: UserWarning: You are saving your model as an HDF5 file via `model.save()`. This file format is considered legacy. We recommend using instead the native Keras format, e.g. `model.save('my_model.keras')`.

110/111 [============================>.] - ETA: 0s - loss: 0.6239 - mean_absolute_error: 0.0249
111/111 [==============================] - 10s 95ms/step - loss: 0.6237 - mean_absolute_error: 0.0250 - val_loss: 0.6305 - val_mean_absolute_error: 0.0245
Epoch 10/30
 54/111 [=============>................] - ETA: 1s - loss: 0.6254 - mean_absolute_error: 0.0241
/home/jean-sebastien/Documents/s7/PMC/venv/lib/python3.11/site-packages/keras/src/engine/training.py:3103: UserWarning: You are saving your model as an HDF5 file via `model.save()`. This file format is considered legacy. We recommend using instead the native Keras format, e.g. `model.save('my_model.keras')`.

110/111 [============================>.] - ETA: 0s - loss: 0.6235 - mean_absolute_error: 0.0240
111/111 [==============================] - 4s 40ms/step - loss: 0.6234 - mean_absolute_error: 0.0240 - val_loss: 0.6302 - val_mean_absolute_error: 0.0231
Epoch 11/30

110/111 [============================>.] - ETA: 0s - loss: 0.6233 - mean_absolute_error: 0.0233
/home/jean-sebastien/Documents/s7/PMC/venv/lib/python3.11/site-packages/keras/src/engine/training.py:3103: UserWarning: You are saving your model as an HDF5 file via `model.save()`. This file format is considered legacy. We recommend using instead the native Keras format, e.g. `model.save('my_model.keras')`.
  saving_api.save_model(
111/111 [==============================] - 4s 38ms/step - loss: 0.6233 - mean_absolute_error: 0.0233 - val_loss: 0.6301 - val_mean_absolute_error: 0.0226
Epoch 12/30

110/111 [============================>.] - ETA: 0s - loss: 0.6229 - mean_absolute_error: 0.0228
/home/jean-sebastien/Documents/s7/PMC/venv/lib/python3.11/site-packages/keras/src/engine/training.py:3103: UserWarning: You are saving your model as an HDF5 file via `model.save()`. This file format is considered legacy. We recommend using instead the native Keras format, e.g. `model.save('my_model.keras')`.
  saving_api.save_model(
111/111 [==============================] - 4s 37ms/step - loss: 0.6231 - mean_absolute_error: 0.0228 - val_loss: 0.6299 - val_mean_absolute_error: 0.0221
Epoch 13/30

103/111 [==========================>...] - ETA: 0s - loss: 0.6231 - mean_absolute_error: 0.0222
/home/jean-sebastien/Documents/s7/PMC/venv/lib/python3.11/site-packages/keras/src/engine/training.py:3103: UserWarning: You are saving your model as an HDF5 file via `model.save()`. This file format is considered legacy. We recommend using instead the native Keras format, e.g. `model.save('my_model.keras')`.
  saving_api.save_model(

111/111 [==============================] - 4s 40ms/step - loss: 0.6230 - mean_absolute_error: 0.0222 - val_loss: 0.6298 - val_mean_absolute_error: 0.0216
Epoch 14/30
 86/111 [======================>.......] - ETA: 0s - loss: 0.6222 - mean_absolute_error: 0.0220
/home/jean-sebastien/Documents/s7/PMC/venv/lib/python3.11/site-packages/keras/src/engine/training.py:3103: UserWarning: You are saving your model as an HDF5 file via `model.save()`. This file format is considered legacy. We recommend using instead the native Keras format, e.g. `model.save('my_model.keras')`.

111/111 [==============================] - ETA: 0s - loss: 0.6228 - mean_absolute_error: 0.0218
111/111 [==============================] - 4s 39ms/step - loss: 0.6228 - mean_absolute_error: 0.0218 - val_loss: 0.6297 - val_mean_absolute_error: 0.0212
Epoch 15/30
 76/111 [===================>..........] - ETA: 0s - loss: 0.6224 - mean_absolute_error: 0.0213
/home/jean-sebastien/Documents/s7/PMC/venv/lib/python3.11/site-packages/keras/src/engine/training.py:3103: UserWarning: You are saving your model as an HDF5 file via `model.save()`. This file format is considered legacy. We recommend using instead the native Keras format, e.g. `model.save('my_model.keras')`.

110/111 [============================>.] - ETA: 0s - loss: 0.6227 - mean_absolute_error: 0.0213
111/111 [==============================] - 4s 39ms/step - loss: 0.6227 - mean_absolute_error: 0.0213 - val_loss: 0.6296 - val_mean_absolute_error: 0.0208
Epoch 16/30
 64/111 [================>.............] - ETA: 1s - loss: 0.6220 - mean_absolute_error: 0.0208
/home/jean-sebastien/Documents/s7/PMC/venv/lib/python3.11/site-packages/keras/src/engine/training.py:3103: UserWarning: You are saving your model as an HDF5 file via `model.save()`. This file format is considered legacy. We recommend using instead the native Keras format, e.g. `model.save('my_model.keras')`.

109/111 [============================>.] - ETA: 0s - loss: 0.6225 - mean_absolute_error: 0.0209
111/111 [==============================] - 4s 39ms/step - loss: 0.6226 - mean_absolute_error: 0.0209 - val_loss: 0.6295 - val_mean_absolute_error: 0.0204
Epoch 17/30

110/111 [============================>.] - ETA: 0s - loss: 0.6226 - mean_absolute_error: 0.0206
/home/jean-sebastien/Documents/s7/PMC/venv/lib/python3.11/site-packages/keras/src/engine/training.py:3103: UserWarning: You are saving your model as an HDF5 file via `model.save()`. This file format is considered legacy. We recommend using instead the native Keras format, e.g. `model.save('my_model.keras')`.
  saving_api.save_model(
111/111 [==============================] - 5s 41ms/step - loss: 0.6225 - mean_absolute_error: 0.0206 - val_loss: 0.6294 - val_mean_absolute_error: 0.0202
Epoch 18/30

103/111 [==========================>...] - ETA: 0s - loss: 0.6225 - mean_absolute_error: 0.0202
/home/jean-sebastien/Documents/s7/PMC/venv/lib/python3.11/site-packages/keras/src/engine/training.py:3103: UserWarning: You are saving your model as an HDF5 file via `model.save()`. This file format is considered legacy. We recommend using instead the native Keras format, e.g. `model.save('my_model.keras')`.
  saving_api.save_model(
111/111 [==============================] - 5s 41ms/step - loss: 0.6225 - mean_absolute_error: 0.0203 - val_loss: 0.6294 - val_mean_absolute_error: 0.0203
Epoch 19/30

 92/111 [=======================>......] - ETA: 0s - loss: 0.6232 - mean_absolute_error: 0.0201
/home/jean-sebastien/Documents/s7/PMC/venv/lib/python3.11/site-packages/keras/src/engine/training.py:3103: UserWarning: You are saving your model as an HDF5 file via `model.save()`. This file format is considered legacy. We recommend using instead the native Keras format, e.g. `model.save('my_model.keras')`.

110/111 [============================>.] - ETA: 0s - loss: 0.6224 - mean_absolute_error: 0.0201
111/111 [==============================] - 4s 40ms/step - loss: 0.6224 - mean_absolute_error: 0.0201 - val_loss: 0.6293 - val_mean_absolute_error: 0.0199
Epoch 20/30
 73/111 [==================>...........] - ETA: 0s - loss: 0.6208 - mean_absolute_error: 0.0199
/home/jean-sebastien/Documents/s7/PMC/venv/lib/python3.11/site-packages/keras/src/engine/training.py:3103: UserWarning: You are saving your model as an HDF5 file via `model.save()`. This file format is considered legacy. We recommend using instead the native Keras format, e.g. `model.save('my_model.keras')`.

109/111 [============================>.] - ETA: 0s - loss: 0.6223 - mean_absolute_error: 0.0198
111/111 [==============================] - 4s 39ms/step - loss: 0.6223 - mean_absolute_error: 0.0198 - val_loss: 0.6292 - val_mean_absolute_error: 0.0195
Epoch 21/30
111/111 [==============================] - 3s 27ms/step - loss: 0.6223 - mean_absolute_error: 0.0195 - val_loss: 0.6292 - val_mean_absolute_error: 0.0196
Epoch 22/30

105/111 [===========================>..] - ETA: 0s - loss: 0.6230 - mean_absolute_error: 0.0195
/home/jean-sebastien/Documents/s7/PMC/venv/lib/python3.11/site-packages/keras/src/engine/training.py:3103: UserWarning: You are saving your model as an HDF5 file via `model.save()`. This file format is considered legacy. We recommend using instead the native Keras format, e.g. `model.save('my_model.keras')`.
  saving_api.save_model(

111/111 [==============================] - 5s 41ms/step - loss: 0.6222 - mean_absolute_error: 0.0195 - val_loss: 0.6292 - val_mean_absolute_error: 0.0200
Epoch 23/30
 83/111 [=====================>........] - ETA: 0s - loss: 0.6208 - mean_absolute_error: 0.0191
/home/jean-sebastien/Documents/s7/PMC/venv/lib/python3.11/site-packages/keras/src/engine/training.py:3103: UserWarning: You are saving your model as an HDF5 file via `model.save()`. This file format is considered legacy. We recommend using instead the native Keras format, e.g. `model.save('my_model.keras')`.

109/111 [============================>.] - ETA: 0s - loss: 0.6221 - mean_absolute_error: 0.0192
111/111 [==============================] - 4s 39ms/step - loss: 0.6222 - mean_absolute_error: 0.0192 - val_loss: 0.6291 - val_mean_absolute_error: 0.0191
Epoch 24/30
 74/111 [===================>..........] - ETA: 0s - loss: 0.6215 - mean_absolute_error: 0.0191
/home/jean-sebastien/Documents/s7/PMC/venv/lib/python3.11/site-packages/keras/src/engine/training.py:3103: UserWarning: You are saving your model as an HDF5 file via `model.save()`. This file format is considered legacy. We recommend using instead the native Keras format, e.g. `model.save('my_model.keras')`.

109/111 [============================>.] - ETA: 0s - loss: 0.6222 - mean_absolute_error: 0.0191
111/111 [==============================] - 4s 38ms/step - loss: 0.6221 - mean_absolute_error: 0.0191 - val_loss: 0.6291 - val_mean_absolute_error: 0.0190
Epoch 25/30
 59/111 [==============>...............] - ETA: 1s - loss: 0.6239 - mean_absolute_error: 0.0187
/home/jean-sebastien/Documents/s7/PMC/venv/lib/python3.11/site-packages/keras/src/engine/training.py:3103: UserWarning: You are saving your model as an HDF5 file via `model.save()`. This file format is considered legacy. We recommend using instead the native Keras format, e.g. `model.save('my_model.keras')`.

111/111 [==============================] - ETA: 0s - loss: 0.6221 - mean_absolute_error: 0.0188
111/111 [==============================] - 4s 40ms/step - loss: 0.6221 - mean_absolute_error: 0.0188 - val_loss: 0.6290 - val_mean_absolute_error: 0.0187
Epoch 26/30

110/111 [============================>.] - ETA: 0s - loss: 0.6218 - mean_absolute_error: 0.0188
/home/jean-sebastien/Documents/s7/PMC/venv/lib/python3.11/site-packages/keras/src/engine/training.py:3103: UserWarning: You are saving your model as an HDF5 file via `model.save()`. This file format is considered legacy. We recommend using instead the native Keras format, e.g. `model.save('my_model.keras')`.
  saving_api.save_model(
111/111 [==============================] - 4s 39ms/step - loss: 0.6220 - mean_absolute_error: 0.0188 - val_loss: 0.6290 - val_mean_absolute_error: 0.0184
Epoch 27/30

109/111 [============================>.] - ETA: 0s - loss: 0.6223 - mean_absolute_error: 0.0186
/home/jean-sebastien/Documents/s7/PMC/venv/lib/python3.11/site-packages/keras/src/engine/training.py:3103: UserWarning: You are saving your model as an HDF5 file via `model.save()`. This file format is considered legacy. We recommend using instead the native Keras format, e.g. `model.save('my_model.keras')`.
  saving_api.save_model(
111/111 [==============================] - 4s 38ms/step - loss: 0.6220 - mean_absolute_error: 0.0186 - val_loss: 0.6290 - val_mean_absolute_error: 0.0184
Epoch 28/30

101/111 [==========================>...] - ETA: 0s - loss: 0.6221 - mean_absolute_error: 0.0185
/home/jean-sebastien/Documents/s7/PMC/venv/lib/python3.11/site-packages/keras/src/engine/training.py:3103: UserWarning: You are saving your model as an HDF5 file via `model.save()`. This file format is considered legacy. We recommend using instead the native Keras format, e.g. `model.save('my_model.keras')`.
  saving_api.save_model(
111/111 [==============================] - 4s 39ms/step - loss: 0.6220 - mean_absolute_error: 0.0185 - val_loss: 0.6289 - val_mean_absolute_error: 0.0186
Epoch 29/30


 87/111 [======================>.......] - ETA: 0s - loss: 0.6233 - mean_absolute_error: 0.0184
/home/jean-sebastien/Documents/s7/PMC/venv/lib/python3.11/site-packages/keras/src/engine/training.py:3103: UserWarning: You are saving your model as an HDF5 file via `model.save()`. This file format is considered legacy. We recommend using instead the native Keras format, e.g. `model.save('my_model.keras')`.

109/111 [============================>.] - ETA: 0s - loss: 0.6222 - mean_absolute_error: 0.0183
111/111 [==============================] - 5s 41ms/step - loss: 0.6219 - mean_absolute_error: 0.0183 - val_loss: 0.6289 - val_mean_absolute_error: 0.0183
Epoch 30/30
 71/111 [==================>...........] - ETA: 0s - loss: 0.6214 - mean_absolute_error: 0.0183
/home/jean-sebastien/Documents/s7/PMC/venv/lib/python3.11/site-packages/keras/src/engine/training.py:3103: UserWarning: You are saving your model as an HDF5 file via `model.save()`. This file format is considered legacy. We recommend using instead the native Keras format, e.g. `model.save('my_model.keras')`.

109/111 [============================>.] - ETA: 0s - loss: 0.6223 - mean_absolute_error: 0.0182

111/111 [==============================] - 4s 39ms/step - loss: 0.6219 - mean_absolute_error: 0.0182 - val_loss: 0.6289 - val_mean_absolute_error: 0.0182